app/webpack.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         wasm_imports: './src/wasm_imports.js',
     },
     output: {
-        path: path.resolve(root,'dist','assets'),
+        path: path.resolve(root,'static','main'),
```

Comment:
Why this change? The generated things should not go to a folder named `static`. It is a widely used convention it goes to `dist`. Moreover, `static/main` is something I would never thought will contain a generated set of JS files.

Reply:
I have tried to put the files to make them accessible in many different locations. Putting it in `src/renderer`, `src/common` or `src/main` doesn't make them easily accessible.

I found out `static` is a path defined by Electron to put static resources, such as binary files, image, etc... And it's fairly well documented. It was the way to go.

Putting the files directly in `static` wasn't a good option either. `index.html`, for instance, gets replaced by a generated `index.html` from Electron's renderer's `index.html`. So that's why I moved them to `static/main`.